### DISCUSSION TITLE
Connect an external button

### DISCUSSION THREAD

**Discussion by guykoy123** [pyControl/code#81] [Q&A] [answered]
I am trying to connect an external physical button. However after checking the docs and the board there doesn't seem to be a space for another Digital Input connection.
Is it possible to connect in some way?
Thanks

**Comment by alustig3** [comment, marked as answer]
You can access the digital inputs by plugging in a [port adapter](https://pycontrol.readthedocs.io/en/latest/user-guide/hardware/#usage) and wire you own button into the screw terminal. You can build your own based on files [here](https://github.com/pyControl/hardware/tree/master/Port_adapter) or purchase one through [open-ephys](https://open-ephys.org/pycontrol/pycontrol-peripherals).

also, there is an external USR button on the pyControl [breakout board](https://github.com/pyControl/hardware/tree/master/Breakout_board) that you can use. There is  `button.py` task in the example folder if you want to try it (or take a look [here](https://github.com/pyControl/code/blob/master/tasks/example/button.py)).
